Pod/Classes/RxRealm.swift

```diff
@@ -172,5 +172,36 @@ public extension Realm {
             }
         }
     }
-
-}
\ No newline at end of file
+    
+    /**
+     Returns bindable sink wich adds objects in sequence to Realm
+     */
+    public func rx_addObjects<S: SequenceType where S.Generator.Element: Object>() -> AnyObserver<S> {
+        return AnyObserver { [weak self] event in
+            switch event {
+            case .Next(let elements):
+                try! self?.write {
+                    self?.add(elements)
+                }
+            default:
+                break
+            }
+        }
+    }
+    
+    /**
+     Returns bindable sink wich adds object to Realm
+     */
+    public func rx_addObject<O: Object>() -> AnyObserver<O> {
+        return AnyObserver { [weak self] event in
+            switch event {
+            case .Next(let object):
+                try! self?.write {
+                    self?.add(object)
+                }
+            default:
+                break
+            }
+        }
+    }
+}
```

Comment:
We can do like `Realm` they have several `add` methods
So we can do the same, kinda:

```Swift
    public func rx_add<S: SequenceType where S.Generator.Element: Object, S: Object>() -> AnyObserver<S> {
        return AnyObserver { [weak self] event in
            switch event {
            case .Next(let items):
                try! self?.write {
                    self?.add(items)
                }
            default:
                break
            }
        }
    }
    
    public func rx_add<O: Object>() -> AnyObserver<O> {
        return AnyObserver { [weak self] event in
            switch event {
            case .Next(let object):
                try! self?.write {
                    self?.add(object)
                }
            default:
                break
            }
        }
    }

```